security/providers/oidc-common/src/main/java/io/helidon/security/providers/oidc/common/BaseBuilder.java

```diff
@@ -444,7 +444,7 @@ public B optionalAudience(boolean optional) {
      * @param checkAudience whether the audience claim will be checked ({@code true}) or not ({@code false})
      * @return updated builder instance
      */
-    @ConfiguredOption("false")
+    @ConfiguredOption("true")
```

Comment:
This change is a backward incompatible behavior

Reply:
It is just a doc fix. Does not change anything in functionality itself.